Packs/Workday/Integrations/Workday_IAM/README.md

```diff
@@ -20,5 +20,8 @@ For more information, please refer to the [Identity Lifecycle Management article
 | first_run | Sync user profiles on first run | False |
 | fetch_samples | Fetch Samples | False |
 | workday_date_format | Date Format in Workday Report | True |
+| deactivation_date_field | Date Format in Workday Report | True |
+| days_before_hire_to_sync | Number of days before hire date to sync hires | True |
+| days_before_hire_to_enable_ad | Number of days before hire date to enable Active Directory account | True |
```

Comment:
lets change the table here
Parameter will be the paramter name (display name)
The description will describe the field/how to configure it/how it will impact on the fetched incidents